brayns/Brayns.cpp

```diff
@@ -273,29 +275,192 @@ struct Brayns::Impl : public PluginAPI
     {
         auto& registry = _engine->getScene().getLoaderRegistry();
         auto& scene = _engine->getScene();
-        auto& geometryParameters = _parametersManager.getGeometryParameters();
 
-        registry.registerLoader(
-            std::make_unique<ProteinLoader>(scene, geometryParameters));
-        registry.registerLoader(std::make_unique<VolumeLoader>(
-            scene, _parametersManager.getVolumeParameters()));
+        registry.registerLoader(std::make_unique<ProteinLoader>(scene));
+        registry.registerLoader(std::make_unique<RawVolumeLoader>(scene));
+        registry.registerLoader(std::make_unique<MHDVolumeLoader>(scene));
         registry.registerLoader(std::make_unique<XYZBLoader>(scene));
 #if BRAYNS_USE_ASSIMP
-        registry.registerLoader(
-            std::make_unique<MeshLoader>(scene, geometryParameters));
+        registry.registerLoader(std::make_unique<MeshLoader>(scene));
 #endif
 #if BRAYNS_USE_LIBARCHIVE
         registry.registerLoader(
             std::make_unique<ArchiveLoader>(scene, registry));
 #endif
-        registry.registerLoader(
-            std::make_unique<MolecularSystemReader>(scene, geometryParameters));
+        registry.registerLoader(std::make_unique<MolecularSystemReader>(scene));
     }
 
+    PropertyMap _createPropertyMap()
```

Comment:
It will be a follow up commit. I had moved the command line arguments to the plugin, but the the commits were extremely conflicting, so we decided to postpone my changes until this PR gets merged.